packages/tree-widget/src/components/TreeWidgetComponent.scss

```diff
@@ -13,7 +13,7 @@
   flex-direction: column;
 
   .components-selectable-content {
-    display: flex;
+    display: block;
     flex-direction: column;
```

Comment:
FYI - no need for flex-direction if display is block.

Reply:
ah good catch bill, i just looked at the changed line. will fix whenever theres a new update to the pkg